content/200-orm/200-prisma-client/100-queries/090-raw-database-access/050-raw-queries.mdx

```diff
@@ -54,6 +56,14 @@ const result = await prisma.$queryRaw(
 )
 ```
 
+<Admonition type="warning">
+
+If you use string building to incorporate untrusted input into queries passed to this method, then you open up the possibility for SQL injection attacks. SQL injection attacks can expose your data to modification or deletion.<br /><br />
```

Comment:
Can we express here what the opposite of "use string building" is? How can one describe the safe way to use this method?